src/FixedProductMarketMakerMapping.ts

```diff
@@ -212,6 +248,14 @@ export function handleFundingAdded(event: FPMMFundingAdded): void {
   setLiquidity(fpmm as FixedProductMarketMaker, newAmounts, collateralScaleDec, collateralUSDPrice)
 
   fpmm.save();
+
+  recordFPMMLiquidity(fpmm as FixedProductMarketMaker, 
+    LIQUIDITY_TYPE_ADD,
+    event.params.amountsAdded,
+    event.params.funder.toHexString(),
+    event.params.sharesMinted,
+    new BigInt(0),
```

Comment:
The constructor is not documented, so use `BigInt.fromI32(0)` instead.

Reply:
Done :)